Upload intern blog post

arceacereza opened the pull request:
uploaded markdown file for Elizabeth's blog post

arceacereza committed:
Add files via upload

uploaded markdown file for Elizabeth's blog post

arceacereza committed:
Update 2022-12-01-Stumbling-Into-Greatness.md

added front matter with correct info

p-galligan commented on _posts/2022-12-01-Stumbling-Into-Greatness.md:
```diff
@@ -0,0 +1,27 @@
+---
+title: "Stumbling-Into-Greatness"
+date: 2023-03-273T10:00:00
```
@arceacereza This PR is failing tests because the date is incorrect.

Can you change this from `2023-03-273` to `2023-03-24`?

arceacereza replied:
Thanks @p-galligan! Just fixed the date. Should we change the document title to match as well?

p-galligan replied:
No, it should be fine.

p-galligan reviewed (changes requested):
Hey, I just have a small comment on this.

p-galligan commented on _posts/2022-12-01-Stumbling-Into-Greatness.md:
```diff
@@ -0,0 +1,27 @@
+
+excerpt_separator: <!--more-->
+---
+
+
+
```
Remove some of these extraneous lines.

p-galligan commented on _posts/2022-12-01-Stumbling-Into-Greatness.md:
```diff
@@ -0,0 +1,27 @@
+
+
+I was told about an internship opportunity with the Rockefeller Archive Center by my roommate. She encouraged me to apply, knowing that I was looking for new opportunities. I applied, was interviewed, and began to work with the RAC in the Fall of 2022. I had no previous experience in archival work, but was excited nonetheless. I was ready to explore more of what you could do with a History degree ...
+
```
I'd say add the excerpt separator here.

`<!--more-->`

arceacereza replied:
@p-galligan Does there need to be a space before/after the separator? Can it be placed right after the paragraph, then add a space before the next paragraph?

p-galligan replied:
I'd say an empty line on either side of it for code legibility reasons. Thanks!

p-galligan commented on _posts/2022-12-01-Stumbling-Into-Greatness.md:
```diff
@@ -0,0 +1,27 @@
+I was told about an internship opportunity with the Rockefeller Archive Center by my roommate. She encouraged me to apply, knowing that I was looking for new opportunities. I applied, was interviewed, and began to work with the RAC in the Fall of 2022. I had no previous experience in archival work, but was excited nonetheless. I was ready to explore more of what you could do with a History degree ...
+
+I worked on site at the Archives at CCNY, with Professor Sydney Van Nort as my supervisor there. It is so nice to be involved with Archives that are part of your college - there is a personal connection. Professor Van Nort helped to acclimate me to the collections. Aside from exploring our collections, my favorite thing that I worked on during my internship was creating outreach for the CCNY Arch ...
+In our digital age, social media is the perfect way for them to engage. I was able to develop my pitching skills for this project. It was something I greatly believed in and wanted to convince my superiors to go for it. Being able to pitch something is a very valuable skill, and I’m glad I was able to flex that muscle.
```
Add a line between the paragraphs

arceacereza committed:
Update 2022-12-01-Stumbling-Into-Greatness.md

fixed date that was causing failed check

p-galligan commented on _posts/2022-12-01-Stumbling-Into-Greatness.md:
```diff
@@ -0,0 +1,27 @@
+I worked on site at the Archives at CCNY, with Professor Sydney Van Nort as my supervisor there. It is so nice to be involved with Archives that are part of your college - there is a personal connection. Professor Van Nort helped to acclimate me to the collections. Aside from exploring our collections, my favorite thing that I worked on during my internship was creating outreach for the CCNY Arch ...
+In our digital age, social media is the perfect way for them to engage. I was able to develop my pitching skills for this project. It was something I greatly believed in and wanted to convince my superiors to go for it. Being able to pitch something is a very valuable skill, and I’m glad I was able to flex that muscle.
+
+Additionally, another thing I worked on was inventory of a small collection. For the first time, I was able to be hands on with items in the Archive. I worked on an alumni file for Valdemar Monshaw. I went through the entire 4 box collection, and cataloged what was in it. Itis
```
Theres a misspelling on this line. "Itis" should be "It is".

p-galligan commented:
@arceacereza sorry, I added a few small comments on formatting/grammar.

arceacereza committed:
Update 2022-12-01-Stumbling-Into-Greatness.md

Removed extraneous lines.
Added excerpt separator after first paragraph.
Added lines in between paragraphs and fixed weird line breaks.
Fixed "Itis" to "It is."

arceacereza commented:
> @arceacereza sorry, I added a few small comments on formatting/grammar.

Thanks, @p-galligan! Something weird was happening with the line breaks and I fixed those. I made it through the rest of the changes you requested.

p-galligan committed:
clean up title

minor changes to title and formatting
